test_cases/example_2.json

```diff
@@ -1,1164 +1,925 @@
+                        "@type": "ElectionResults.InternationalizedText"
+                    },
+                    "@type": "ElectionResults.Candidate"
+                },
+                {
+                    "@id": "candidate_alpha",
+                    "BallotName": {
+                        "Text": [
+                            {
+                                "Content": "Anthony Alpha",
+                                "Language": "en",
+                                "@type": "ElectionResults.LanguageString"
+                            }
+                        ],
+                        "@type": "ElectionResults.InternationalizedText"
+                    },
+                    "IsTopTicket": true,
+                    "PartyId": "lepton_party",
+                    "@type": "ElectionResults.Candidate"
+                },
+                {
+                    "@id": "candidate_gamma",
+                    "BallotName": {
+                        "Text": [
+                            {
+                                "Content": "Gloria Gamma",
+                                "Language": "en",
+                                "@type": "ElectionResults.LanguageString"
+                            }
+                        ],
+                        "@type": "ElectionResults.InternationalizedText"
+                    },
+                    "IsTopTicket": true,
+                    "PartyId": "hadron_party",
+                    "@type": "ElectionResults.Candidate"
+                },
+                {
+                    "@id": "candidate_beta",
+                    "BallotName": {
+                        "Text": [
+                            {
+                                "Content": "Betty Beta",
+                                "Language": "en",
+                                "@type": "ElectionResults.LanguageString"
+                            }
+                        ],
+                        "@type": "ElectionResults.InternationalizedText"
+                    },
+                    "PartyId": "lepton_party",
+                    "@type": "ElectionResults.Candidate"
+                },
+                {
+                    "@id": "candidate_delta",
+                    "BallotName": {
+                        "Text": [
+                            {
+                                "Content": "David Delta",
+                                "Language": "en",
+                                "@type": "ElectionResults.LanguageString"
+                            }
+                        ],
+                        "@type": "ElectionResults.InternationalizedText"
+                    },
+                    "PartyId": "hadron_party",
+                    "@type": "ElectionResults.Candidate"
+                }
+            ],
+            "BallotStyle": [
+                {
+                    "@type": "ElectionResults.BallotStyle",
+                    "GpUnitIds": ["precinct_1_downtown"],
```

Comment:
Worth mentioning that this is the only ballot style. Is the goal of this example to provide a single ballot style, rather than multiple ballot styles in a single EDF?

That completely makes sense, but wanted to explicitly call it out to make sure I'm on the same page.

Reply:
I don't see that; there are four objects with "@type": "ElectionResults.BallotStyle" in the BallotStyle array.

Reply:
You are correct; disregard.